admin/assets/javascripts/global-scripts.js

```diff
@@ -253,6 +253,26 @@ $(function () {
     }
   }
 
+  /**
+   * Sticky banner positioning.
+   */
+  function stickyBannerPositioning () {
+    var windowHeight = $(window).height()
```

Comment:
The admin app should be functional without javascript.  What happens here?

Reply:
@jon-shipley We are going for JS enabled only, I've confirmed this three times.

Reply:
I see. That's fine then.